HWB.Rmd

```diff
@@ -0,0 +1,144 @@
+---
+title: "HW5B_Angela_Chun"
+author: "Angela Chun"
+date: "8/3/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyr)
+library(dplyr)
+library(readr)
+library(ggplot2)
+library(stringr)
+```
+
+
+# Load the data
+```{r load data}
+AUC_1 <- read.csv("CTRP_files/AUC_1.csv")
+AUC_2 <- read.csv("CTRP_files/AUC_2.csv")
+cancer_cell_line_info <- read.csv("CTRP_files/cancer_cell_line_info.csv")
+compound_info <- read.csv("CTRP_files/compound_info.csv")
+experiment_info <- read.csv("CTRP_files/Experiment_info.csv")
+```
+
+
+# Join the data 
+```{r joined data}
+#joining all the data into one data frame
+Joined_CTRP_data <- AUC_1 %>%
+  bind_rows(AUC_2) %>%
+  inner_join(experiment_info, by = c("experiment_id" = "expt_id")) %>%
+  inner_join(cancer_cell_line_info, by = "master_ccl_id")%>%
+  inner_join(compound_info, by = "master_cpd_id") 
+
+```
+
+
+
+
+# Answering Questions
+
+
+### Q1) Which cancer type has the lowest AUC values to the compound "vorinostat"?
+```{r Q1}
+
+#finding the cancer_type with lowest mean AUC
+CTRP_VOR <- Joined_CTRP_data %>%
+  filter(cpd_name == "vorinostat") %>%
+  group_by(cancer_type) %>%
+  summarize(AUC = mean(area_under_curve)) %>%
+  arrange(AUC)
+  
+#visualizing the data 
+CTRP_VOR
+
+#Answer = haematopoietic_and_lymphoid_tissue
```

Comment:
I agree, great job, and I learnt of using group_by and summarize.

I may have misunderstand this question, I sorted for the lowest AUC value acorss all samples, instead of the mean value for each cancer type, and then sort